### PR TITLE
Add images for UG

### PR DESCRIPTION
**Pull request by D-Limiter**

### CODE REVIEW

**Review by WangCheng0116 — CHANGES_REQUESTED** [review]
It should be from /images under /docs, not under docs/ directly, otherwise images won't appear in UG. And another thing is once you update your forked repo, there's no need to close the pr and start again, github will run the tests again automatically for you

**Comment by D-Limiter** [comment]
> It should be from /images under /docs, not under docs/ directly, otherwise images won't appear in UG. And another thing is once you update your forked repo, there's no need to close the pr and start again, github will run the tests again automatically for you

i have made the respective changes, thanks!

File: docs/ListDuringReview.png
Comment (itsNatTan) [inline docs/ListDuringReview.png:1]: Delete

File: docs/ListSuccess.png
Comment (itsNatTan) [inline docs/ListSuccess.png:1]: Delete

File: docs/ReviewOver.png
Comment (itsNatTan) [inline docs/ReviewOver.png:1]: Delete

File: docs/ReviewSuccess.png
Comment (itsNatTan) [inline docs/ReviewSuccess.png:1]: Delete

File: docs/Stats.png
Comment (itsNatTan) [inline docs/Stats.png:1]: Delete

**Review by itsNatTan — CHANGES_REQUESTED** [review]
Delete

File: docs/ListDuringReview.png
Comment (D-Limiter) [reply docs/ListDuringReview.png:1]: Hv deleted the required files

File: docs/ListSuccess.png
Comment (D-Limiter) [reply docs/ListSuccess.png:1]: Hv deleted the required files

**Review by WangCheng0116 — APPROVED** [review]
Looks good!

**Review by itsNatTan — APPROVED** [review]
Don't understand why there was a need to remove images with 5 commits, but lgtm

**Pull request merged**